components/nut/Makefile

```diff
@@ -0,0 +1,67 @@
+#
```

Comment:
oops, thanks

2016-03-30 20:23 GMT+02:00 Alexander Pyhalov <notifications@github.com>:

> In components/nut/Makefile
> <https://github.com/OpenIndiana/oi-userland/pull/1821#discussion_r57938141>
> :
>
> > @@ -0,0 +1,67 @@
> > +#
>
> nut doesn't belong to this PR
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1821/files/ba58e172fa68ceea48b9cb72ee2c69c1afda192b#r57938141>
>